1/Dockerfile.rhel7

```diff
@@ -47,13 +47,10 @@ ADD ./contrib/s2i /usr/libexec/s2i
 # NOTE: When adding new Jenkins plugin, you have to create the symlink for the
 # HPI file created by rpm to /opt/openshift/plugins folder.
 RUN mkdir -p /opt/openshift/plugins && \
-    ln -s /usr/lib64/jenkins/openshift-pipeline.hpi /opt/openshift/plugins/openshift-pipeline.jpi && \
-    ln -s /usr/lib64/jenkins/credentials.hpi /opt/openshift/plugins/credentials.jpi && \
     # need to create <plugin>.pinned files when upgrading "core" plugins like credentials or subversion that are bundled with the jenkins server
+    for FILENAME in /usr/lib64/jenkins/* ; do ln -s $FILENAME /opt/openshift/plugins/`basename $FILENAME .hpi`.jpi; done &&\
```

Comment:
/usr/lib64/jenkins/*hpi

if we have jpi files in there too, you'll need another command since you don't want to end up with foo.jpi.jpi as the symlink.